tests/fsharp/NUnitHelpers.fs

```diff
@@ -1,17 +1,44 @@
-namespace NUnit.Framework
+namespace Xunit
+
+open Xunit
+open Xunit.Sdk
+open System.Threading
+open System.Globalization
+
+type UseCulture(culture: string, uiCulture: string) =
```

Comment:
If you have a minute for this, try just removing this altogether, if the test passes locally and in the CI. 

If no, don't bother and keep this code, I just remember that I removed that when moving some other tests and nothing happened.